pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -229,7 +229,12 @@ def __transfer_distrib_spawn_state_on_fit_end(self, trainer: "pl.Trainer", resul
                 self.mp_queue.put(best_model_path)
                 self.mp_queue.put(last_path)
                 self.mp_queue.put(results)
-                self.lightning_module.add_to_queue(self.mp_queue)  # adds the `callback_metrics` to the queue
+                # adds the `callback_metrics` to the queue
+                # TODO: Remove the if in v1.7
+                if is_overridden("add_to_queue", self.lightning_module):
```

Comment:
Let's add an example for the `add_to_queue` example in the documentation. Ideally, Optuna with best metrics.

Reply:
in this PR? 

also related: #10865

Reply:
noted your request in the linked issue